main.tf

```diff
@@ -186,11 +186,11 @@ module "acm" {
 
 locals {
   mount_path = "/home/atlantis"
-  mount_points = var.enable_efs ? [{
+  mount_points = var.enable_efs ? concat(try(var.atlantis.mount_points, []), [{
```

Comment:
I think this is such an edge case that if you want to do it, you might as well just provide all of the mount points (and therefore, no change required here)

Reply:
In case if  `enable_efs` is `true` - EFS volume and acces points are manged in the atlantis module, but `var.atlantis.mount_points` is ignored - it is not possible to provide all mount points as you suggested.

The only option would be to set `enable_efs` to `false` and manage EFS elsewhere.